wagtail/wagtailcore/blocks/field_block.py

```diff
@@ -261,8 +261,23 @@ def __init__(self, required=True, help_text=None, max_length=None, min_length=No
             widget=forms.Textarea)
         super(RawHTMLBlock, self).__init__(**kwargs)
 
-    def render_basic(self, value):
-        return mark_safe(value)  # if it isn't safe, that's the site admin's problem for allowing raw HTML blocks in the first place...
+    def get_default(self):
+        return mark_safe(self.meta.default or '')
+
+    def to_python(self, value):
+        return mark_safe(value)
+
+    def get_prep_value(self, value):
+        # explicitly convert to a plain string, just in case we're using some serialisation method
+        # that doesn't cope with SafeText values correctly
+        return str(value)
```

Comment:
Good point! `str()` does indeed fail on non-ASCII characters in Py2 - fixed + tested in 60bafb3.